src/app/static-pages/landing-page/sections/newsletter-section/newsletter-section.component.html

```diff
@@ -0,0 +1,31 @@
+<h1>Newsletter</h1>
+
+<div class="wrapper">
+  <div class="content">
+    <div class="text">
+      <img class="image" height="110" src="assets/images/illustrations/undraw_Mailbox_re_dvds.svg" alt="Briefkasten" />
+      <h2>Bleib' auf dem Laufenden</h2>
+      <p>Erhalte News, Events, Download-Tipps und vieles mehr. Direkt von codetekt!</p>
```

Comment:
Außerdem würde ich den Text ändern in "Abonniere unseren Newsletter, um über alle wichtigen Entwicklungen rund um unsere Trust-Checking Plattform und den Verein codetekt e.V., informiert zu werden!"